processor/src/test/java/org/mapstruct/ap/testutil/assertions/JavaFileAssert.java

```diff
@@ -38,6 +38,8 @@
         "\"\\d{4}-\\d{2}-\\d{2}T\\d{2}:\\d{2}:\\d{2}\\+\\d{4}\",";
     private static final String GENERATED_COMMENTS_REGEX = "\\s+comments = \"version: , compiler: .*, environment: " +
         ".*\"";
+    private static final String IMPORT_GENERATED_ANNOTATION_REGEX = "import javax\\.annotation\\.(processing\\.)" +
+        "?Generated;";
```

Comment:
Really really minor enhancement in case you still need to change something else: 
Move the `?` in the previous line. I just read this line several times and was wondering when a class/package name could have `?` in it until I realized this `?` is the usual reg-exp quantifier... :rofl:

Reply:
Changed it